src/seismometer/_api.py

```diff
@@ -348,13 +348,27 @@ def _cohort_list_details(cohort_dict: dict[str, tuple[Any]]) -> HTML:
     groups = data[cfg.entity_keys + cfg.output_list + intervention_cols + outcome_cols + target_cols].groupby(
         target_cols
     )
-    aggregation = {cfg.entity_id: ["count", "nunique"]}
+    columns = []
+    for c in ["count", "nunique"]:
```

Comment:
This loop is a bit unexpected,  pd.Groupby.agg() can usually handle lists of agg functions.
Can you look into passing a dict in, and then overwriting column names?

The gist:
```python
stat_dict = {k: ['mean'] for k in list(data.select_dtypes(include=float))}
stat_dict.update({cfg.entity_id: ['nunique', 'count'], 
                  cfg.context_id: ['nunique'], ...})
groupstats = groups.agg(stat_dict)
groupstats.columns = [' '.join(c) for c in groupstats.columns]
```

Reply:
I was running into an crash based on 
![image](https://github.com/user-attachments/assets/75724236-e4e1-4aaf-9984-71dc21e4a5cb)

So the kludge was to deal with not being able to pass a dictionary. 
I looked into the core code and applied the right fix (thanks for this comment)